active/af3-resynthesize-phi.py

```diff
@@ -111,6 +112,7 @@ def populate_blobid_in_job_table(**kwargs):
                       "VALUES (%s, %s, %s, %s, %s) ".format(table=common_variables.AF3_RUNS_DETAILS))
 
     for creation_date in datecreated:
+      try:
```

Comment:
Would you move this to line 123? Generally I'd like to see us make our 'try' blocks as narrow in scope as possible.

Reply:
absolutely - thanks for seeing that.

Reply:
*edit*
this is where the call to SQL Server is - line 123 is the call to the Postgres database, which we haven't seemed to have issues with

Reply:
is there another place you see to put the try catch for the sql call?